### ISSUE TITLE
Pink header Bar with FreeREG logo etc too wide for small screens

### ISSUE THREAD

**Issue by dreams-togo** [FreeUKGen/ComETT#129]
On small screens this header bar is too wide. If you make the logo a bit smaller it can be reduced to around 50% of it&#039;s current depth as the text is quite small. This is an assessiblity issue as every inch of space for the image and transcription fields is precious. This is a design issue with assessibility in mind

**Comment by stsccfr** [comment]
I agree that, in general, transcribers will want to maximise the view of the image and the transcription, and minimise everything else.  Even on a desktop screen, I still want to see as much of the image as I can, subject only to my ability to read the text, and I also want to see as many transcribed records as possible within the remaining space.  When I am transcribing a post-1836 printed Anglican register, I arrange things so that I can see 1 complete marriage record or 4 complete baptism or burial records.  In the screen space that is left I can usually fit at least 8 transcribed records.
